docs/running-on-mesos.md

```diff
@@ -167,6 +167,14 @@ acquire. By default, it will acquire *all* cores in the cluster (that get offere
 only makes sense if you run just one application at a time. You can cap the maximum number of cores
 using `conf.set("spark.cores.max", "10")` (for example).
 
+# Mesos Docker Support
+
+Spark can make use of a Mesos Docker containerizer by setting the property `spark.mesos.executor.docker.image`
+in your [SparkConf](configuration.html#spark-properties)
```

Comment:
Can you add "Note that this requires Mesos version X or later" here instead? Then you don't need to duplicate that line in all the relevant configs.